Support NginxCisConnector resource (#1286)
nginxinc/kubernetes-ingress: deployments/helm-chart/README.md

`controller.service.type` | The type of service to create for the Ingress controller. | LoadBalancer
`controller.service.externalTrafficPolicy` | The externalTrafficPolicy of the service. The value Local preserves the client source IP. | Local
`controller.service.annotations` | The annotations of the Ingress controller service. | {}

> `controller.service.extraLabels` | The extra labels of the service. | {}

Reviewer: Are these extra labels that are already in use by an existing service? Or extra labels to add to a Service that we're deploying?

Author: @jputrino extra labels to add to a Service that we're deploying

Reviewer: ```suggestion
`controller.service.extraLabels` | Defines extra labels to add to the service. | {}
```

`controller.service.httpsPort.targetPort` | The target port of the HTTPS port of the Ingress controller service. | 443
`controller.serviceAccount.name` | The name of the service account of the Ingress controller pods. Used for RBAC. | Autogenerated
`controller.serviceAccount.imagePullSecretName` | The name of the secret containing docker registry credentials. Secret must exist in the same namespace as the helm release. | ""

> `controller.reportIngressStatus.enable` | Update the address field in the status of Ingresses resources with an external address of the Ingress controller. You must also specify the source of the external address either through an external service via `controller.reportIngressStatus.externalService`, `controller.reportIngressStatus.nginxCisConnector` or the `external-status-address` entry in the ConfigMap via `controller.config.entries`. **Note:** `controller.config.entries.external-status-address` takes precedence over the others. | true

Reviewer: ```suggestion
`controller.reportIngressStatus.enable` | Updates the address field in the status of Ingresses resources with an external address of the Ingress controller. You must also specify the source of the external address either through an external service via `controller.reportIngressStatus.externalService`, `controller.reportIngressStatus.nginxCisConnector` or the `external-status-address` entry in the ConfigMap via `controller.config.entries`. **Note:** `controller.config.entries.external-status-address` takes precedence over the others. | true
```

> `controller.reportIngressStatus.enable` | Updates the address field in the status of Ingresses resources with an external address of the Ingress controller. You must also specify the source of the external address either through an external service via `controller.reportIngressStatus.externalService`, `controller.reportIngressStatus.nginxCisConnector` or the `external-status-address` entry in the ConfigMap via `controller.config.entries`. **Note:** `controller.config.entries.external-status-address` takes precedence over the others. | true

Reviewer: Should this be "Ingress resources" instead of "Ingresses resources"?

`controller.reportIngressStatus.externalService` | Specifies the name of the service with the type LoadBalancer through which the Ingress controller is exposed externally. The external address of the service is used when reporting the status of Ingress, VirtualServer and VirtualServerRoute resources. `controller.reportIngressStatus.enable` must be set to `true`. The default is autogenerated and enabled when `controller.service.create` is set to `true` and `controller.service.type` is set to `LoadBalancer`. | Autogenerated

> `controller.reportIngressStatus.nginxCisConnector` |  Specifies the name of the NginxCisConnector resource, which exposes the Ingress Controller pods via a BIG-IP. The IP of the BIG-IP is used when reporting the status of Ingress, VirtualServer and VirtualServerRoute resources. `controller.reportIngressStatus.enable` must be set to `true`. | ""

Reviewer: ```suggestion
`controller.reportIngressStatus.nginxCisConnector` |  Specifies the name of the NginxCisConnector resource, which exposes the Ingress Controller pods via a BIG-IP system. The IP of the BIG-IP system is used when reporting the status of Ingress, VirtualServer and VirtualServerRoute resources. `controller.reportIngressStatus.enable` must be set to `true`. | ""
```
